backup-client/main.go

```diff
@@ -101,11 +100,6 @@ func createK8sClient(namespace string) (client.Client, error) {
 }
 
 func doBackup(ctx context.Context, name, addr string) error {
-	if medusaClient, err := medusa.NewClient(addr); err != nil {
-		return err
-	} else {
-		defer medusaClient.Close()
-		return medusaClient.CreateBackup(ctx, name)
-	}
+	//
```

Comment:
If CreateBackup is no longer called, why does this method exists?

Reply:
`backup-client` is not actually used. I only left it for reference. `CreateBackup` is used [here](http://github.com/k8ssandra/medusa-operator/blob/4bbbcd55d7fbe4d75acd79c9ac18705f75f22494/controllers/cassandrabackup_controller.go#L261-L261)